### PR TITLE
refactor(console): remove add user subtitle

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

remove add user subtitle and name requirement

## Testing

image

### CODE REVIEW

File: packages/console/src/pages/Users/components/CreateForm/index.tsx [hunk 1]
```diff
@@ -44,7 +44,6 @@ const CreateForm = ({ onClose }: Props) => {
   return (
     <ModalLayout
       title="users.create"
-      subtitle="users.subtitle"
```
Comment (IceHe) [inline packages/console/src/pages/Users/components/CreateForm/index.tsx:47]: should remove the related description in en.ts & zh-cn.ts

**Review by IceHe — APPROVED** [review]
Overall LGTM

File: packages/console/src/pages/Users/components/CreateForm/index.tsx (hunk 1 above)
Comment (gao-sun) [reply packages/console/src/pages/Users/components/CreateForm/index.tsx:47]: it's used for user mgmt page

Comment (demonzoo) [reply packages/console/src/pages/Users/components/CreateForm/index.tsx:47]: No, we still need this subtitle in `Users` page

**Review by demonzoo — APPROVED** [review]

**Pull request merged**